x-pack/plugins/apm/server/routes/latency_distribution/get_percentile_threshold_value.ts

```diff
@@ -6,32 +6,35 @@
  */
 
 import { CommonCorrelationsQueryParams } from '../../../common/correlations/types';
-import { ProcessorEvent } from '../../../common/processor_event';
+import { LatencyDistributionChartType } from '../../../common/latency_distribution_chart_types';
 import { Setup } from '../../lib/helpers/setup_request';
 import { fetchDurationPercentiles } from '../correlations/queries/fetch_duration_percentiles';
 
 export async function getPercentileThresholdValue({
   setup,
-  eventType,
+  chartType,
   start,
   end,
   environment,
   kuery,
   query,
   percentileThreshold,
+  searchMetrics,
 }: CommonCorrelationsQueryParams & {
   setup: Setup;
-  eventType: ProcessorEvent;
+  chartType: LatencyDistributionChartType;
   percentileThreshold: number;
+  searchMetrics?: boolean;
```

Comment:
Can you make this required? It should be easy to pass in `true` or `false`, I'd rather be explicit about it then forget about it somewhere.